2_process/src/aggregate_observations.R

```diff
@@ -35,11 +35,13 @@ combine_wq_data <- function(wqp_data, nwis_data){
               Value_Max_cd = NA_character_,
               Value_Min = min(resultVal2, na.rm = TRUE),
               Value_Min_cd = NA_character_,
+              #using max to select 'u', which is greater than 'd'.
+              data_type = max(data_type),
```

Comment:
This is a clever use of `max()`. Just to check my own understanding - are there actually any instances of 'u' in the WQP data? Based on the definition in `p2_wqp_SC_filtered` I assumed all of these would be 'd'.

Reply:
They are all 'u' for WQP, so I can remove the max here. I copied this code from the combined daily + instantaneous function (also edited in this PR)